corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail_hide_data_table.xml

```diff
@@ -50,7 +50,7 @@
           <series nodeset="instance('reports')/reports/report[@id='ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i']/rows/row[@is_total_row='False']">
             <configuration/>
             <x function="column[@id='owner']" />
-            <y function="column[@id='tickets']" />
+            <y function="column[@id='count']" />
```

Comment:
why did renaming this fix the test?

Reply:
I think the report was originally incorrect. there was no column called tickets. I think the author meant to use the column "count"
https://github.com/dimagi/commcare-hq/pull/30873/commits/079bbc6196469f52af88769111b3339588138230#diff-e07a4c4fece2663ad926ae268445b50fe989a1b0b4183e5e9e79ab74f75e8d58
```
{
    "type": "field",
    "display": {
        "en": "Tickets",
        "fr": "Billets"
    },
    "description": "Number of tickets",
    "field": "count",
    "aggregation": "sum"
}
```